spring-boot-autoconfigure/src/test/java/org/axonframework/springboot/autoconfig/AxonServerAutoConfigurationTest.java

```diff
@@ -48,4 +79,69 @@ public void testAxonServerQueryBusConfiguration() {
         assertTrue(queryBus instanceof AxonServerQueryBus);
         assertSame(updateEmitter, queryBus.queryUpdateEmitter());
     }
-}
\ No newline at end of file
+
+    @Test
+    public void testAxonServerCommandBusBeanTypesConfiguration() {
+        assertTrue(commandBus instanceof AxonServerCommandBus);
+        assertTrue(localSegment instanceof SimpleCommandBus);
+    }
+
+    @Test
+    public void testAxonServerDefaultCommandBusConfiguration() {
+        this.contextRunner
```

Comment:
Sweet, didn't know about this feature. Makes the unit tests more concise I'd say.